__mocks__/@react-navigation/native/index.ts

```diff
@@ -1,9 +1,65 @@
-import {useIsFocused as realUseIsFocused, useTheme as realUseTheme} from '@react-navigation/native';
+/* eslint-disable import/prefer-default-export, import/no-import-module-exports */
+import type * as ReactNavigation from '@react-navigation/native';
+import createAddListenerMock from '../../../tests/utils/createAddListenerMock';
 
-// We only want these mocked for storybook, not jest
-const useIsFocused: typeof realUseIsFocused = process.env.NODE_ENV === 'test' ? realUseIsFocused : () => true;
+const isJestEnv = process.env.NODE_ENV === 'test';
```

Comment:
When would this mock get used in an environment other than a jest test?

Reply:
I think it is the storybook tests, @roryabraham can you confirm?

Reply:
yep we use mocks with storybook too.